doc/nrf/working_with_nrf/nrf53/thingy53.rst

```diff
@@ -299,37 +290,17 @@ FOTA updates on the Nordic Thingy:53 are supported using MCUmanager's Simple Man
 The application acts as a GATT server and allows the connected Bluetooth Central to perform the firmware update for both the application and network core.
 The Bluetooth configuration is updated to allow quick DFU data transfer.
 
-The application supports SMP handlers related to:
-
-* Image management.
-* Operating System (OS) management -- used to reboot the device after the firmware upload is complete.
-* Erasing settings partition -- used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
-
-To enable the MCUmanager library and handling of SMP commands over Bluetooth, use the options :kconfig:option:`CONFIG_MCUMGR` and :kconfig:option:`CONFIG_MCUMGR_SMP_BT`.
-Also, disable encryption and authentication requirement for the Bluetooth SMP transport using :kconfig:option:`CONFIG_MCUMGR_SMP_BT_AUTHEN`.
-
-You need to enable the MCUmanager handlers for image and OS management.
-Use the options :kconfig:option:`CONFIG_MCUMGR_CMD_IMG_MGMT` and :kconfig:option:`CONFIG_MCUMGR_CMD_OS_MGMT`.
-
-You also have to enable the MCUmanager processing of basic Zephyr group commands and storage erase commands.
-Use the options :kconfig:option:`CONFIG_MCUMGR_GRP_ZEPHYR_BASIC` and :kconfig:option:`CONFIG_MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE`.
-
-See :file:`thing53_nrf5340_cpuapp.conf` in a compatible sample for how this is done.
-
-In addition, the application must call the functions :c:func:`os_mgmt_register_group` and :c:func:`img_mgmt_register_group` to register the image and OS management command handlers.
-The application must also call :c:func:`smp_bt_register` to register the SMP Bluetooth Service.
-
-See the :file:`thingy53.c` source file in a compatible sample for how this is done.
+To enable FOTA feature, use :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option.
+For details about implementation, please take a look at :ref:`ug_nrf52_developing_ble_fota` guide.
```

Comment:
I would also note that Thingy:53 supports network core upgrade by default (not supported for other nRF53 boards, because only Thingy:53 enables it in its MCUboot bootloader's default configuration).